sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -482,24 +487,23 @@ export class ApplicationService extends RecordDataModelService<Application> {
         "programYear.endDate",
         "sinValidation.id",
         "sinValidation.sin",
-        "studentAssessments.id",
-        "studentAssessments.triggerType",
+        "currentAssessment.id",
       ])
       .innerJoin("application.programYear", "programYear")
       .leftJoin("application.pirProgram", "pirProgram")
       .innerJoin("application.student", "student")
       .innerJoin("student.sinValidation", "sinValidation")
       .innerJoin("application.location", "location")
-      .innerJoin("application.studentAssessments", "studentAssessments")
-      .leftJoin("studentAssessments.offering", "offering")
+      .innerJoin("application.currentAssessment", "currentAssessment")
+      .leftJoin("currentAssessment.offering", "offering")
       .leftJoin("offering.educationProgram", "educationProgram")
       .innerJoin("student.user", "user")
       .leftJoin("application.pirDeniedReasonId", "PIRDeniedReason")
       .where("application.id = :applicationId", {
         applicationId,
       })
       .andWhere("location.id = :locationId", { locationId })
-      .andWhere("studentAssessments.triggerType = :triggerType", {
+      .andWhere("currentAssessment.triggerType = :triggerType", {
```

Comment:
I am not sure if I am following this. The PIR will have an original assessment at the start but does this query is intended to work also when a PIR assessment is reassessed?